commitlint.config.js

```diff
@@ -1 +1,26 @@
-module.exports = { extends: ['@commitlint/config-conventional'] };
+// @see https://github.com/conventional-changelog/commitlint/blob/v12.0.1/@commitlint/config-conventional/index.js
+module.exports = {
+    extends: ['@commitlint/config-conventional'],
+
+    rules: {
+        'body-max-line-length': [1, 'always', 100],
+        'header-max-length': [1, 'always', 100],
```

Comment:
After checking `institute-plugin` with previously suggested rules I noticed that header lines are usually longer than 72 chars. Also checked GitHub length again and it's 70, so even shorter.

With that in mind and also @lkraav preference to do longer, I decided on using upstream length but making them warnings instead of errors.

```diff
- 'body-max-line-length': [2, 'always', 100],
+ 'body-max-line-length': [1, 'always', 100],

- 'header-max-length': [2, 'always', 100],
+ 'header-max-length': [1, 'always', 100],
```